index.html

```diff
@@ -968,17 +951,23 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 whether a resource's timing information can be shared with the
 <a>current document</a>, is as follows:</p>
 <ol>
-<li>
-<p>If the resource is same origin, return <code>pass</code>.</p>
-</li>
-<li>
-<p>If the <a>Timing-Allow-Origin</a> header value list contains a
-case-sensitive match for the value of the <code><a data-cite=
-"RFC6454#section-4" data-lt="http-origin">origin</a></code> of the
-<a>current document</a>, or a wildcard ("<code>*</code>"), return
-<code>pass</code>.</p>
-</li>
-<li>Return <code>fail</code>.</li>
+    <li><p>Let <var>tainted</var> be <code>false</code>.</p></li>
+    <li><p>For each request in the resource <a
+    data-cite="FETCH#concept-fetch">fetch</a> redirection-chain:</p>
+        <ol>
+            <li><p>If the <a>Timing-Allow-Origin</a> header value list
+            does not contain a value which is byte-for-byte identical to
+            the <a
+            data-cite="HTML#ascii-serialisation-of-an-origin">serialization</a>
+            of the <a>current document</a>'s <a
+            data-cite="DOM#concept-document-origin">origin</a>, nor a
+            wildcard ("<code>*</code>"), and the request is
+            <a>cross-origin</a> or <var>tainted</var> is
+            <code>true</code>, return <code>fail</code>.</p></li>
+            <li><p>If the request is <a>cross-origin</a>, set
+            <var>tainted</var> to <code>true</code>.</p></li>
```

Comment:
I'd move this up one step, so the next step only has to check tainted and becomes a little more readable. I'd also suggest to not use `<code>` for prose booleans.

Reply:
done!